cmd/virtual-workspaces/command/cmd.go

```diff
@@ -188,6 +193,18 @@ func Run(ctx context.Context, o *options.Options) error {
 		return err
 	}
 
+	// apply admission to check permission claims for resources through a view's URL
+	admissionPluginInitializers := []admission.PluginInitializer{
+		kcpadmissioninitializers.NewKcpInformersInitializer(wildcardKcpInformers, cacheKcpInformers),
+	}
+	o.CoreVirtualWorkspaces.Admission.ApplyTo(
+		&recommendedConfig.Config,
+		informerfactoryhack.Wrap(wildcardKubeInformers),
+		clientsethack.Wrap(localShardKubeClusterClient),
+		utilfeature.DefaultFeatureGate,
+		admissionPluginInitializers...,
+	)
+
```

Comment:
as commented below, this block looks misplaced. It's not special for the standalone command.

Reply:
Agreed, this was from Nolan and me figuring out what was needed to make things work. Now we'll make them correct 😄. Thanks for the tips.